promscale/troubleshooting.md

```diff
@@ -0,0 +1,90 @@
+# Troubleshooting Promscale
+This section covers some common errors or problems you might run into while using
+ Promscale.
+
+<!---
+* Keep this section in alphabetical order
+* Use this format for writing troubleshooting sections:
+ - Cause: What causes the problem?
+ - Consequence: What does the user see when they hit this problem?
+ - Fix/Workaround: What can the user do to fix or work around the problem? Provide a "Resolving" Procedure if required.
+ - Result: When the user applies the fix, what is the result when the same action is applied?
+* Copy this comment at the top of every troubleshooting page
+-->
+
+## Backing up Promscale data gives an `out of shared memory` error
```

Comment:
```suggestion
## Backing up Promscale data gives an "out of shared memory" error
```

@jacobprall - will quotes work here? Inline ticks to not (unless we have another workaround). With back ticks, the heading becomes "Backing up Promscale data gives an"

Reply:
@ryanbooz the quotes aren't working in the header, so dropped them from this PR. Here is the rendered image with`` in header.

![Screenshot 2022-02-25 at 11 40 27 PM](https://user-images.githubusercontent.com/25104868/155766567-158bef05-bf17-420e-ab00-e526ddb188c9.png)